webui/src/main/java/org/polypheny/db/webui/Crud.java

```diff
@@ -703,6 +709,96 @@ Result insertRow( final Request req, final Response res ) {
     }
 
 
+    /**
+     * Gets the classified Data from User
+     * return possibly interesting Data to User
+     * @return
+     */
+
+    public Result classifyData( Request req, Response res ) {
+        ClassifyAllData classifyAllData = this.gson.fromJson( req.body(), ClassifyAllData.class );
+        ExploreManager e = ExploreManager.getInstance();
+
+        Explore explor = e.classifyData( classifyAllData.id, classifyAllData.classified );
+        return new Result( classifyAllData.header, explor.getData() );
+    }
+
+    public Result createQuery(Request req, Response res){
+
+        QueryExplorationRequest queryExplorationRequest = this.gson.fromJson( req.body(), QueryExplorationRequest.class );
+        ExploreManager exploreManager = ExploreManager.getInstance();
+        Transaction transaction = getTransaction( queryExplorationRequest.analyze );
+        boolean autoCommit = true;
+
+        ArrayList<Result> results = new ArrayList<>();
+        Result result;
+        long executionTime = 0;
+        long temp = 0;
+
+        String query = queryExplorationRequest.query;
+        List<String> group = new ArrayList<>(  );
+
+        Map<String, String> typeInfo = new HashMap<>(  );
+
+        try {
+            Result r = executeSqlSelect( transaction, queryExplorationRequest, query ).setInfo( new Debug().setGeneratedQuery( query ) );
+
+            for ( int i = 0; i < r.getHeader().length; i++ ){
+                typeInfo.put( r.getHeader()[i].name, r.getHeader()[i].dataType );
+               group.add( r.getHeader()[i].dataType );
+            }
+
+        } catch (  QueryExecutionException | RuntimeException e ) {
+            e.printStackTrace();
```

Comment:
Please remove printStackTrace() calls.

Reply:
resolved in commit:[https://github.com/polypheny/Polypheny-DB/pull/211/commits/5aa89f0c4a4106cfc45536101cf1877310f5dea9](url)